### PR TITLE
Add postcode validation for Slovenia

### PR DESCRIPTION
**Pull request by R33D3M33R**
Postcode in Slovenia consists of 4 numbers. First number can be in range 1-9 and the rest can also include 0.

https://en.wikipedia.org/wiki/Postal_codes_in_Slovenia

### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changelog entry

Add postcode validation for Slovenia

### CODE REVIEW

File: includes/class-wc-validation.php [hunk 1]
```diff
@@ -96,7 +96,9 @@ public static function is_postcode( $postcode, $country ) {
 			case 'NL':
 				$valid = (bool) preg_match( '/^([1-9][0-9]{3})(\s?)(?!SA|SD|SS)[A-Z]{2}$/i', $postcode );
 				break;
-
+			case 'SI':
+				$valid = (bool) preg_match( '/^([1-9][0-9]{3})$/i', $postcode );
```
Comment (rodrigoprimo) [inline includes/class-wc-validation.php]: I believe the `i` modifier is not necessary as this expression matches only numbers?

**Review by rodrigoprimo — COMMENTED** [review]
Thanks for your contribution. Overall looks good to me and it works. I left just one comment with a suggestion of a change before we merge this.

File: includes/class-wc-validation.php (hunk 1 above)
Comment (R33D3M33R) [reply includes/class-wc-validation.php]: Hello, you are right. I overlooked this when modifying the NL case.

**Review by rodrigoprimo — APPROVED** [review]
Thanks for updating the PR. Looks good now.

**Pull request merged**